File: CleanFix/WebApi/Models/CompanyDto.cs
namespace WebApi.Models;

public class CompanyDto
{
    public int Id { get; set; } // Identificador único de la empresa
    public string? Name { get; set; }
    public string? Address { get; set; }
    public string? Number { get; set; }
    public string? Email { get; set; }
    public string? Type { get; set; } // Ejemplo: "Electricidad", "Fontanería", etc.
    public decimal? Price { get; set; }
    public int? WorkTime { get; set; }

}

File: CleanFix-frontend/src/app/core/infrastructure/repositories/company/CompanyDto.ts
export interface CompanyDto {
  id: number
  name: string
  address: string
  number: string
  email: string
  type: number
  price: number
  workTime: number
}

export enum IssueType {
  Fontaneria,
  Electricidad,
  Carpinteria,
  Pintura,
  Suelos,
  Limpieza,
  Listo,
}
